src/resources/commandTemplates/templates.ActionCommand.ts

```diff
@@ -565,7 +606,7 @@ export default class ActionCommand extends ResponsiveSlashCommandSubcommandBuild
     }
 
     await sendToLogChannel(interaction.client, USER, LOG, action[3]);
-    if (action[3].sendNoticeFirst) await sendNotice(USER, LOG, interaction);
+    if (action[3].sendNoticeFirst && !action[3].noNotice) await sendNotice(USER, LOG, interaction);
```

Comment:
We should consider having some sort of test/assertion sendnoticefirst and nonotice aren't both true- if they ever *are* true someone made a mistake

Reply:
Done. It will also throw an error (that will not stop the bot) to make sure we see it.

Reply:
Alright, I think this is fine. // TODO to get some other test because this should be something we can check (at compile time even?) and if this misconfiguration ever does occur I believe the action will currently become impossible to perform